force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -291,7 +420,13 @@ public with sharing class VariablesDataTypesOperators {
      * @return A Decimal representing the monthly paycheck.
      */
     public static Decimal calculateMonthlyPaycheck(Decimal hourlyRate, Double numberOfHours) {
-        return null; // Replace null with the variable you used to store the result
+        If (hourlyRate == null || numberOfHours == null)
+            { 
+            return null;
+               }
+          Decimal weeklyPayCheck = (hourlyRate * Decimal.valueOf(numberOfHours)).setScale(2, RoundingMode.HALF_UP);
+         Decimal monthlyPayCheck = (weeklyPayCheck * 4).setScale(2, RoundingMode.HALF_UP);
+        return monthlyPayCheck;
```

Comment:
Although correct,
Is there a simpler way you can do this without so many different methods?